openprescribing/measure_definitions/probiotics.json

```diff
@@ -8,11 +8,12 @@
   ],
   "why_it_matters": [
     "The Advisory Committee on Borderline Substances (ACBS) is responsible for assessing and recommending products applying for listing in Part XV of the Drug Tariff. The ACBS recently reviewed some products listed in Part XV to ensure they are still clinically effective and appropriate to be listed here. They considered the probiotics VSL#3 and Vivomixx and advised the Secretary of State for Health and Social Care to remove these products. The Secretary of State accepted this recommendation and removed the products from the drug tariff in November 2018.",
-    "The Committee concluded that the evidence did not sufficiently demonstrate that the products are clinically effective. Additionally",
+    "This is consistent with <a href='https://www.england.nhs.uk/medicines/conditions-for-which-over-the-counter-items-should-not-routinely-be-prescribed/'> NHS England guidance </a> that recommends probitoics should nolonger be rotuinely prescribed.",
+    "<p> The Committee concluded that the evidence did not sufficiently demonstrate that the products are clinically effective. Additionally",
```

Comment:
You've added a `<p>` here, but there's no closing `</p>`.  You should probably also have `<p>`...`</p>` around the first paragraph too.

Reply:
I'll fix this now.